src/test/java/shouty/CoordinateTest.java

```diff
@@ -0,0 +1,14 @@
+package shouty;
+
+import junit.framework.TestCase;
+import org.junit.Test;
+
+public class CoordinateTest extends TestCase {
+
+    @Test
+    public void testItCalculatesTheDistanceFromAnotherCoordinateAlongXAxis() {
```

Comment:
In IDEA: Right-click method, class file or dir and select run. Make note of keyboard shortcut and do it with keyboard next time.